### ISSUE TITLE
@ParameterizedTest skipped when invoked from method level 

### ISSUE THREAD

**Issue by jbelford** [Microsoft/vscode-java-test#526]
@ParameterizedTest works when run from the class level but I run into errors when executed from the method level.

```
WARNING: TestEngine with ID 'junit-jupiter' failed to discover tests
org.junit.platform.commons.util.PreconditionViolationException: Could not find method with name [myArgumentMethod] in class [org.example.ExampleTest].
	at org.junit.platform.engine.discovery.MethodSelector.lambda$lazyLoadJavaMethod$2(MethodSelector.java:177)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at org.junit.platform.engine.discovery.MethodSelector.lazyLoadJavaMethod(MethodSelector.java:175)
	at org.junit.platform.engine.discovery.MethodSelector.getJavaMethod(MethodSelector.java:142)
	at org.junit.jupiter.engine.discovery.DiscoverySelectorResolver.lambda$resolveSelectors$3(DiscoverySelectorResolver.java:62)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.jupiter.engine.discovery.DiscoverySelectorResolver.resolveSelectors(DiscoverySelectorResolver.java:61)
	at org.junit.jupiter.engine.JupiterTestEngine.resolveDiscoveryRequest(JupiterTestEngine.java:68)
	at org.junit.jupiter.engine.JupiterTestEngine.discover(JupiterTestEngine.java:61)
	at org.junit.platform.launcher.core.DefaultLauncher.discoverEngineRoot(DefaultLauncher.java:130)
	at org.junit.platform.launcher.core.DefaultLauncher.discoverRoot(DefaultLauncher.java:117)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:90)
	at com.microsoft.java.test.runner.junit5.CustomizedConsoleTestExecutor.lambda$executeTests$0(CustomizedConsoleTestExecutor.java:42)
	at com.microsoft.java.test.runner.junit5.CustomContextClassLoaderExecutor.invoke(CustomContextClassLoaderExecutor.java:29)
	at com.microsoft.java.test.runner.junit5.CustomizedConsoleTestExecutor.executeTests(CustomizedConsoleTestExecutor.java:38)
	at com.microsoft.java.test.runner.junit5.CustomizedConsoleLauncher.execute(CustomizedConsoleLauncher.java:29)
	at com.microsoft.java.test.runner.Launcher.main(Launcher.java:53)
```

**Comment by jdneo** [comment]
@jbelford Thanks for reporting. Looks like this is caused by this issue: #134

And we are planning to solve it in the next version: 0.13.0

**Comment by jdneo** [comment]
This should be fixed with #539
